src/backend/metadata/tmdb.ts

```diff
@@ -155,13 +156,32 @@ const headers = {
 async function get<T>(url: string, params?: object): Promise<T> {
   if (!apiKey) throw new Error("TMDB API key not set");
 
-  const res = await mwFetch<any>(encodeURI(url), {
-    headers,
-    baseURL,
-    params: {
-      ...params,
-    },
-  });
+  let res: Promise<T>;
```

Comment:
We're not returning a promise, we're returning the value itself as the promises are being awaited

```suggestion
  let res: T;
```